<!--
///
/// \file task4.ui
/// \brief XML-описание интерфейса окна задач по математическому анализу
/// \details Содержит описание следующих элементов интерфейса:
/// - 4 блока задач по матанализу с полями для ввода решений
/// - Кнопки проверки решений для каждой задачи
/// - Поля отображения результатов проверки (✓ Верно/✗ Неверно)
/// - Кнопку возврата в главное меню
/// \note Файл автоматически генерируется Qt Designer
/// \warning Ручные правки могут быть перезаписаны при изменении в Qt Designer
/// \see Task4 Класс, реализующий логику работы с данным интерфейсом
///
-->
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task4</class>
 <widget class="QWidget" name="Task4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Исчисление</string>
   </property>
  </widget>
  <widget class="QLabel" name="problemLabel1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="problemLabel2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="problemLabel3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="problemLabel4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerLineEdit1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerLineEdit2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerLineEdit3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerLineEdit4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Вернуться</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultLabel4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
